conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However it's important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should communicate in a professional and respectful manner.
+- Students should use words like "Please" and "Thank you".
+- Students will allow other members and teams speak freely.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not communicate in a way that offends or disrespects others.
+- Students should not interrupt others - we all have something to share.
+- Students should not show signs of passive aggressive behavior. Working as a team means keeping emotions out of work.
```

Comment:
I agree with @mariakimheinert 's assessment of the last part. Maybe something like that working as a team means "keeping a professional tone" with regard to development work.